module/src/utils.c

```diff
@@ -6,6 +6,7 @@
 *
 * @date 04/11/2018
 */
+#include <linux/inet.h>
```

Comment:
@cppcoffee This should actually be moved inside `parser.h` so that the fuzzer still works.

Should be here: https://github.com/landhb/DrawBridge/blob/master/module/include/parser.h#L18

Reply:
Moved in https://github.com/landhb/DrawBridge/pull/28/commits/dc2201a055a7c83c6cfae3852cb4dd4417ca2daf